UI.tsv

```diff
@@ -325,7 +325,7 @@ UI_20150317_000324	{@st56}Dead{/}
 UI_20150317_000325	Spell Power
 UI_20150317_000326	Defense
 UI_20150317_000327	Magic Resistance
-UI_20150317_000328	Critical Rate
+UI_20150317_000328	Critical Attack Chance
```

Comment:
Guild Wars + Guild Wars 2 use 'Critical Chance'
Runescape calls it 'Critical Hit Chance'
Tera Online calls it 'Crit Rate'
Wildstar calls it 'Critical Hit Chance'
World of Warcraft calls it 'Critical Strike Chance'
Elder Scrolls Online calls it 'Critical Chance'


---

Critical Rate to me means the rate at which critical hits will happen. Though the stat screen in game is awful, it's hard to know what '45' could be. Is it an index or is it a flat chance?

@sunaries @ttgmichael @ArkKnight 

I am definitely FOR keeping it as Critical Rate or Critical Chance.